test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/CreateFilesBeforeReconnect.java

```diff
@@ -73,7 +73,7 @@ private synchronized HapiSpecOperation generateFileCreateOperation() {
 	}
 
 	private HapiApiSpec runCreateFiles() {
-		PerfTestLoadSettings settings = new PerfTestLoadSettings(10, 2, 1);
+		PerfTestLoadSettings settings = new PerfTestLoadSettings(6, 3, 1);
```

Comment:
It will be better to have private constants that describes names `6, 3, 1`, as the meaning of numbers is difficult to understand here.

Reply:
Agreed and fixed. Thanks!